src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -429,10 +426,11 @@ def load_node_run_info_for_line(self, line_number: int = None) -> List[NodeRunIn
                 runs = self._load_info_from_file(node_run_record_file)
                 if runs:
                     run = runs[0]
-                run = resolve_multimedia_data_recursively(node_run_record_file, run)
-                run = load_multimedia_data_recursively(run)
-                run_info = NodeRunInfo.deserialize(run)
-                node_run_infos.append(run_info)
+                    multimedia_processor = MultimediaProcessor.create(run.get("message_format", ""))
```

Comment:
Need ming to help check this indent change. @guming-learning

Reply:
The original code should be buggy. Please go ahead.